ci/travis/test.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+echo "library path", $LD_LIBRARY_PATH
```

Comment:
Sure but I don't understand why sqlite does not see it ?
I should investigate on a docker based on your osgeo4travis configuration ...